landing-page/styles.css

```diff
@@ -0,0 +1,122 @@
+*{
+  margin:0;
```

Comment:
There should be a space between the colon and the following value -- looks like you have some with spaces below and some not.

Reply:
Same with the selector -- should be a space between it and the opening brace